cmd/manager/aggregator.go

```diff
@@ -542,6 +542,20 @@ func createResults(crClient aggregatorCrClient, scan *compv1alpha1.ComplianceSca
 		return nil
 	}
 
+	// This feels like an appropriate seam to implementing forwarding since
+	// we have a list of consistent results and remediations. Determine if
+	// we're configured to forward results to an external system and
+	// establish that connection here.
+	//
+	// f := getForwardingClient()
+	//
+	// The getForwardingClient() function is a factory that returns the
+	// appropriate forwarding client based on the configuration supplied in
+	// the `ScanSetting`. This should keep forwarding implementation
```

Comment:
Just curious, is it only a ScanSetting field or can it be applied individually to a Suite/Scan? (my guess no on the latter)

Reply:
That's an interesting though. If it only applies to a ScanSetting, then I can reuse that ScanSetting for all scans I want to forward, right?

For example, if I have a `grpc-forwarder-scan-setting`, my results will be forwarded so long as they use that scan setting. But if I use the `default` scan setting, they won't. Is that understanding correct?

Reply:
ComplianceScanSetting has options for storage and proxy, so it makes sense for it to be there and apply for scans that are bound..